src/main/java/net/pms/dlna/MapFile.java

```diff
@@ -75,6 +78,18 @@ public MapFile(MapFileConfiguration conf, List<File> list) {
 		forcedName = null;
 	}
 
+	protected boolean isFolderThumbnail(File file) {
```

Comment:
Can the name of this function be more clear, or a doc added to explain what it is checking? I don't know at a glance if it is reporting whether the folder has a thumbnail, whether it should have one, whether it has the default one, whether it has a custom one, etc.